src/store/academy-context.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { createContext, ReactNode } from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import type { ImageDataLike } from 'gatsby-plugin-image'
+
+type AcademyDataType = {
```

Comment:
nit: can we change  type names to this format: `TYourTypeName`? for instance here : `TAcademyData`
I see that in deriv com we also use this format.

Reply:
@shayan-deriv the current format for deriv-com is **ComponentNameType** or **ComponentNameProps**